### DISCUSSION TITLE
Edit color choice

### DISCUSSION THREAD

**Discussion by ke352802081770314** [iSoron/uhabits#960] [Feature Requests]
I am sure regarding the goal of the app this is a side issue, but still: I am missing some colors for coloring my habits. The default choice is lacking brown, for example, and has some colors that are so similar they give me no added value (bottom left two look black, and top right two are almost the same orange). I see two solutions:
a) have a settings dialogue that allows me to customize the choice (replace a position in the selection by a different color selected from a color picker), or
b) have one "color" in the selection when creating/editing a habit that opens a color picker for any color I want for that habit. This could be symbolized by a circle with three vertical dots.

**Comment by end-ichi** [comment]
I would prefer option b although we could also have both. The color choices might also just be a sort of history of the colors you previously picked (like in GIMP color picker).

I want to add that in the color picker I really want to be able to input a hex code. I have a system for coloring my calendar and I want to be able to use those same colors in the habits app.

Also I don't get why choosing a color results in two different colors for in-app text/graphs and widget color. That should be the same IMO.

**Comment by awwRoar** [comment]
Please add an option to define hex values for choosing the color. The default choice of colours feel very limited.

**Comment by AbelHutten** [comment]
I would also really appreciate the ability to choose colors via hex code.
